github/pulls_reviews.go

```diff
@@ -50,3 +62,49 @@ func (s *PullRequestsService) ListReviews(owner string, repo string, number int)
 
 	return *reviews, resp, err
 }
+
+// GetReview fetches the specified pull request review.
+//
+// GitHub API docs: https://developer.github.com/v3/pulls/reviews/#get-a-single-review
+func (s *PullRequestsService) GetReview(owner string, number int, id int) (*PullRequestReview, *Response, error) {
+	u := fmt.Sprintf("repos/%v/pulls/%d/reviews/%d", owner, number, id)
```

Comment:
GitHub API documentation has a mistake for this endpoint and the one below. They're both missing the `repo` argument.

I've tested the API, and confirmed it's a documentation issue.

For example, this works okay:

```
curl -i -H "Accept: application/vnd.github.black-cat-preview+json" 'https://api.github.com/repos/google/go-github/pulls/497/reviews'
```

One if the IDs it returns is `13441488`. Then, if we use GitHub API docs to get that single URL, without the repo parameter, it's a 404:

```
$ curl -i -H "Accept: application/vnd.github.black-cat-preview+json" 'https://api.github.com/repos/google/pulls/497/reviews/13441488'
HTTP/1.1 404 Not Found

{
  "message": "Not Found",
  "documentation_url": "https://developer.github.com/v3"
}
```

But if we include repo (as one would expect... it makes no sense to specify a PR with just owner and PR number, without repo name), it works:

```
$ curl -i -H "Accept: application/vnd.github.black-cat-preview+json" 'https://api.github.com/repos/google/go-github/pulls/497/reviews/13441488'
HTTP/1.1 200 OK

{
  "id": 13441488,
  ...
}
```